proxy/handler_one2many.go

```diff
@@ -247,9 +247,8 @@ func (s *handler) forwardClientsToServerMultiStreaming(sources []backendConnecti
 						if err != nil {
 							return s.sendError(src, dst, err)
 						}
-						if err := dst.SetHeader(md); err != nil {
-							return fmt.Errorf("error setting headers from client %s: %w", src.backend, err)
-						}
+
+						dst.SetHeader(md) //nolint:errcheck // ignore errors, as we might try to set headers multiple times
```

Comment:
this is the actual change, everything else is just unrelated Kres updates/linting fixes :exclamation: